### ISSUE TITLE
The possibility of modifying the checkpoint and reloading the model parameter

### ISSUE THREAD

**Issue by peteryang1031** [EleutherAI/pythia#173]
Hi, thank you for developing such a useful tool! 
My requirement is that, modify the parameter of the trained model (checkpoint) and then reload it to the models to get the final performance. Is it possible to do that in Pythia package? 
Looking forward to your kind response.

**Comment by StellaAthena** [comment]
Yes. I do wish to clarify that Pythia isn't a package, and the recommended inference library is HF's transformers. You can do this in transformers just like you can with any other model.
